standalone-metastore/src/main/sql/mysql/hive-schema-2.3.0.mysql.sql

```diff
@@ -833,99 +829,132 @@ CREATE TABLE IF NOT EXISTS `KEY_CONSTRAINTS`
 
 CREATE INDEX `CONSTRAINTS_PARENT_TABLE_ID_INDEX` ON KEY_CONSTRAINTS (`PARENT_TBL_ID`) USING BTREE;
 
-CREATE INDEX `CONSTRAINTS_CONSTRAINT_TYPE_INDEX` ON KEY_CONSTRAINTS (`CONSTRAINT_TYPE`) USING BTREE;
```

Comment:
any particular reason why these lines need to be removed?

Reply:
Looking at the patch created by git apply it took both hive-schema-3.0.0 and hive-schema-2.3.0 to be copies of the same file.  I think it is showing this as removed because they aren't in hive-schema-2.3.0 (which they shouldn't be).  We'll have to be careful when we check this in that everything lands in the right place.  I checked the actual code in my branch and it looks right.